app/build.gradle

```diff
@@ -68,8 +68,8 @@ android {
         applicationId = 'com.loafwallet'
         minSdkVersion 29
         targetSdkVersion 34
-        versionCode 20241118
-        versionName "v2.12.2"
+        versionCode 20241130
+        versionName "v2.12.4"
```

Comment:
Why do we add 2 in every our new app version?

Reply:
Wondering, is there an odd-even release strategy?

Reply:
I'll message directly

Reply:
sure, thanks!